### PR TITLE
Bump calypso-build to 1.0.0-rc.3. Remove file-loader and Babel config.

### PR DESCRIPTION
**Pull request by jeffersonrabb**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Mirroring changes in https://github.com/Automattic/newspack-blocks/pull/7, this PR bumps `@automattic/calypso-build` to 1.0.0-rc.3, removes an unnecessary dependency (`file-loader`), and removes the Babel config file. 

Closes # .

### How to test the changes in this Pull Request:

`npm ci && npm run build:webpack`

Verify no errors, then navigate to `/admin.php?page=newspack-subscriptions-wizard` and verify two `Card` components one with a simple checkbox and one with a checkbox with tooltip. 

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +56,7 @@
   "scripts": {
     "build:style-plugins-screen": "node-sass assets/plugins-screen/plugins-screen.scss assets/plugins-screen/plugins-screen.css --output-style expanded && postcss -r assets/plugins-screen/plugins-screen.css",
     "build:style-plugins-screen-rtl": "rtlcss assets/plugins-screen/plugins-screen.css assets/plugins-screen/plugins-screen-rtl.css",
-    "build:webpack": "webpack",
+    "build:webpack": "calypso-build --config='./webpack.config.js'",
```
Comment (jeffersonrabb) [inline package.json]: @ockham is this change necessary? I copied directly from similar change you made in `newspack-blocks`, but here the build seems to work with or w/out this change.

Comment (ockham) [reply package.json]: Without the `--config` option, `calypso-build` will use its own `webpack.config.js` -- which in this case means it'll use an entry point (default is `src/index`) that's different from the ones this project's `webpack.config.js` is specifying.
